tests/test_client_regress/tests.py

```diff
@@ -1268,6 +1268,17 @@ def test_empty_string_data(self):
         response = self.client.head('/body/', data='', content_type='application/json')
         self.assertEqual(response.content, b'')
 
+    def test_json(self):
+        # Regression test for #24773
```

Comment:
I don't think a reference to the ticket is really needed (only when there is some additional context provided by the ticket that cannot be captured in comments or a docstring).